StripeConnect/StripeConnect/Source/Internal/StripeConnectConstants.swift

```diff
@@ -15,11 +15,10 @@ enum StripeConnectConstants {
      - Open popups in PopupWebViewController (instead of Safari)
      */
     static let allowedHosts: Set<String> = [
-        "connect-js.stripe.com",
```

Comment:
No longer needed since this is the default base url. We have logic to add the base url to the allowed host list.

Reply:
Also added tests to cover this.